packages/snap/src/index.ts

```diff
@@ -143,29 +141,62 @@ async function sendTransaction(tx: CeloTransactionRequest, wallet: CeloWallet) {
   return txResponse.wait()
 }
 
-async function getBIP44Node(index: number = 0): Promise<BIP44Node> {
-  // Get the bip44 node corresponding to the path m/44'/52752'/0'/0
-  const bip44Node = await snap.request({
+async function getKeyPair(snap: SnapsGlobalObject, address?: string, addressIndex: number = 0): Promise<KeyPair> {
+  const derivationPath = "m/44'/60'/0'/0"; //Todo - read from state config
```

Comment:
when we use the bip44 code `52752` a different address is generated, but 60 works fine.

Reply:
i think 60 is what we want, its what viem does for instance https://github.com/wagmi-dev/viem/blob/main/src/accounts/hdKeyToAccount.ts#L18C14-L18C14

i like the TODO

Reply:
`0xce10` (= `52752`) is the coin type index we claimed for Celo in [SLIP-44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). It's expected the address will be different than when using Ethereum's index (`0x3c` = `60`).

Reply:
That said, it's often useful to allow setting this by the user, as they may be using an address generated for a different chain (usually Ethereum), often because the address was generated using a wallet that's not handling all the different SLIP-44 codes (like MetaMask, a Ledger, etc.).

Reply:
yeah i think most people using metamask will be expecting the address to match the one in metamask UI  which would also be the one the dapp sees. 

Having ability to set to the  coin type index for celo could be helpful for some

Reply:
thanks @m-chrzan  yes you are correct, and thats what we do, give them those during installations. Please see below 
<img width="367" alt="image" src="https://github.com/celo-org/gas-snap/assets/8665738/16a8087b-02e6-4393-9aa5-b5e5c510a308">